deploy/charts/cert-manager/cainjector/values.yaml

```diff
@@ -9,6 +9,7 @@ global:
   priorityClassName: ""
   rbac:
     create: true
+    pspEnabled: false
```

Comment:
It depends on the charts, a lot of chart are using rbac.pspEnabled so that's what I used but I do not specially care `podSecurityPolicy.enabled` is fine.